.coderabbit.yaml

```
@@ -0,0 +1,30 @@
+
+  guidelines:
+    - Verify Manifest V3 compliance
+    - Check permissions scope for "scripting", "storage", and "activeTab"
+    - Review content script injection patterns
+    - Validate cross-origin XHR requests in background scripts
+    - Ensure proper error handling in API calls
+    - Check GDPR compliance for data collection
+    - Verify webpack build optimization
+
+feedback: true
+auto_approve: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)